src/screens/collectibles-home/collectibles-home.tsx

```diff
@@ -53,22 +45,33 @@ interface SocialLinksInterface {
 const SMALL_SOCIAL_ICON_SIZE = formatSize(15);
 
 export const CollectiblesHome = () => {
-  const styles = useCollectiblesHomeStyles();
+  const { navigate } = useNavigation();
   const dispatch = useDispatch();
-  const collections = useCreatedCollectionsSelector();
-  const collectibles = useCollectiblesListSelector();
 
+  const collections = useCreatedCollectionsSelector();
+  const collectibles = useCollectiblesWithFullData();
   const selectedAccount = useSelectedAccountSelector();
   const visibleAccounts = useVisibleAccountsListSelector();
+  const isShowCollectibleInfo = useIsShowCollectibleInfoSelector();
+
+  const styles = useCollectiblesHomeStyles();
   const colors = useColors();
   const { height: windowHeight } = useWindowDimensions();
+
   const [headerHeight, setHeaderHeight] = useState(1);
   const [visibleBlockHeight, setVisibleBlockHeight] = useState(1);
-  const { navigate } = useNavigation();
 
-  const insets = useSafeAreaInsets();
-  const TAB_BAR_HEIGHT = 53 + insets.bottom;
-  const ICON_COVER_GAP = 2;
+  const iosSafeArea = useSafeAreaInsets();
+
+  const androidSafeAreaValue = isDefined(initialWindowMetrics)
+    ? initialWindowMetrics.insets.bottom > 0
+      ? initialWindowMetrics.insets.bottom + initialWindowMetrics.insets.top
+      : initialWindowMetrics.insets.bottom
+    : 0;
```

Comment:
I'm confused of this expression

if `initialWindowMetrics.insets.bottom > 0` we add bottom and top
but if it's false, it means that `initialWindowMetrics.insets.bottom = 0` (it couldn't be negative, right?) , so it's also return as `initialWindowMetrics.insets.bottom` which is 0

so, if I'm right, I suggest this one be better:
```
```
isDefined(initialWindowMetrics) && initialWindowMetrics.insets.bottom > 0 
   ? initialWindowMetrics.insets.bottom + initialWindowMetrics.insets.top : 0
```